scripts/install-windows-prereqs.ps1

```diff
@@ -528,54 +528,45 @@ function Install-DCAP-Dependencies {
         $drivers = @{
             'WinServer2016' = @{
                 'sgx_base_dev' = @{
-                    'zip_path'    = "$PACKAGES_DIRECTORY\Intel_SGX_DCAP\Intel SGX DCAP for Windows *\LC_driver_${OS_VERSION}\Signed_*.zip"
+                    'path'        = "$PACKAGES_DIRECTORY\Intel_SGX_DCAP\Intel*SGX*DCAP*\base\WindowsServer2016"
                     'location'    = 'root\SgxLCDevice'
                     'description' = 'Intel(R) Software Guard Extensions Launch Configuration Service'
                 }
                 'sgx_dcap_dev' = @{
-                    'zip_path'    = "$PACKAGES_DIRECTORY\Intel_SGX_DCAP\Intel SGX DCAP for Windows *\DCAP_INF\${OS_VERSION}\Signed_*.zip"
+                    'path'        = "$PACKAGES_DIRECTORY\Intel_SGX_DCAP\Intel*SGX*DCAP*\dcap\WindowsServer2016"
                     'location'    = 'root\SgxLCDevice_DCAP'
                     'description' = 'Intel(R) Software Guard Extensions DCAP Components Device'
                 }
             }
             'WinServer2019' = @{
                 'sgx_base' = @{
-                    'zip_path'    = "$PACKAGES_DIRECTORY\Intel_SGX_DCAP\Intel SGX DCAP for Windows *\LC_driver_${OS_VERSION}\Signed_*.zip"
+                    'path'        = "$PACKAGES_DIRECTORY\Intel_SGX_DCAP\Intel*SGX*DCAP*\base\WindowsServer2019_Windows10"
                     'location'    = 'root\SgxLCDevice'
                     'description' = 'Intel(R) Software Guard Extensions Launch Configuration Service'
                 }
                 'sgx_dcap' = @{
-                    'zip_path'    = "$PACKAGES_DIRECTORY\Intel_SGX_DCAP\Intel SGX DCAP for Windows *\DCAP_INF\${OS_VERSION}\Signed_*.zip"
+                    'path'        = "$PACKAGES_DIRECTORY\Intel_SGX_DCAP\Intel*SGX*DCAP*\dcap\WindowsServer2019_Windows10"
                     'location'    = 'root\SgxLCDevice_DCAP'
                     'description' = 'Intel(R) Software Guard Extensions DCAP Components Device'
                 }
             }
             'Win10' = @{
```

Comment:
Is the Win10 definition redundant now seeing as it's identical to the WS2019 one? I'm also a little nervous that the Win10 detection in Get-WindowsRelease only covers two specific versions and probably doesn't capture all the new versions since that logic was originally written.

Reply:
Any suggestions on how we should address this?

Reply:
We could just merge them into a single config `Win10_2019`.

Reply:
@BRMcLaren Yeah, I think that should be done, I was more curious about 

> I'm also a little nervous that the Win10 detection in Get-WindowsRelease only covers two specific versions

Reply:
Maybe add this to the use case requirements of this [issue](https://github.com/openenclave/openenclave/issues/3000). Then when we tackle this in the next month or so completely refactor this script to treat less as a silver bullet.

The various windows versions include a server version, and I think we might wanna break that out into the CI use case versus a developer situation.

Reply:
To clarify the comment, I'm specifically referring to this code:

```powershell
function Get-WindowsRelease {
    $releases = @{
        18363 = "Win10"
        18362 = "Win10"
        17763 = "WinServer2019"
        14393 = "WinServer2016"
    }
...
```

It seems like our handling of Win10 is very narrow to those two versions for some reason. Realistically, the only thing that the script seems to care about is whether it's using the RS3 (16299) and up versions of Windows or not, so this script could be made less brittle in its current form by checking for the whether we are above or below that version instead of trying to split this into fixed versions (of which "Win10" spans a very wide range up to the current 19041).

Moving forwards in rethinking the script, I think it's an unscalable proposition to enumerate and test every version of Win10 that a dev could run on anyway, and we would likely simply ensure that it works with the latest Win10 client as a sanity test, with the expectation that the install instructions for WS2019 should apply to it.

Reply:
What is the min version that should be checked?

Reply:
@radhikaj We can probably keep 14393 as the min version; that's Win10 RS1 and also WS2016, IIRC.